force-app/main/default/permissionsets/Admin.permissionset-meta.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PermissionSet xmlns="http://soap.sforce.com/2006/04/metadata">
+    <applicationVisibilities>
+        <application>Khaled</application>
+        <visible>true</visible>
+    </applicationVisibilities>
+    <description>Allows system administrator access to be able to troubleshoot problems in all areas.</description>
```

Comment:
Following on from this, your admin permission set only gives access to your Mainly Digital objects, and because of this your description is invalid. You've used a permission set, you could use it on top of a system admin profile, but seeming as the system admin would have access to all objects anyway, you may as well only use the profile.

Reply:
Complete